### ISSUE TITLE
Sort entries from highest to lowest by stars count in graph's legend

### ISSUE THREAD

**Issue by Yunir** [bytebase/star-history#108]
For now it is hard to follow just by color which repository's name belongs to graph's line.

It would be better to sort them in the legend pane by count of current stars.

**Comment by Yunir** [comment]
It becomes even harder if there will be similar colors or more entries if there won't be any sort.
So sorting would ease gathering information from the graphs faster.
For now there is no sorting. Example:
![image](https://user-images.githubusercontent.com/16368046/147860639-7815cfec-2389-41db-834a-ed00ba30f3bb.png)

**Comment by boojack** [comment]
Thanks for your great advising, it has be released in https://github.com/bytebase/star-history/pull/111
